android/app/src/main/java/app/organicmaps/downloader/MapManager.java

```diff
@@ -208,7 +208,7 @@ public static boolean warn3gAndDownload(Activity activity, final String countryI
     return warnOn3g(activity, countryId, () -> {
       if (onAcceptListener != null)
         onAcceptListener.run();
-      nativeDownload(countryId);
+      startDownload(countryId);
```

Comment:
`nativeUpdate()` calls should be changed similarly so that fg service is started